android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonFormActivity.java

```diff
@@ -1033,6 +1038,30 @@ private void setReadOnlyAndFocus(View view, boolean visible, boolean popup) {
         }
     }
 
+    /**
+     * Gets the {@link AppCompatRadioButton} views on the whole {@link com.vijay.jsonwizard.widgets.NativeRadioButtonFactory} and updates the enabled status
+     *
+     * @param view    {@link View}
+     * @param enabled {@link Boolean}
+     */
+    private void setReadOnlyRadioButtonOptions(View view, boolean enabled) {
+        if (view != null) {
+            for (int i = 0; i < ((RadioGroup) view).getChildCount(); i++) {
+                RelativeLayout relativeLayout = (RelativeLayout) ((RadioGroup) view).getChildAt(i);
+                if (relativeLayout != null) {
```

Comment:
null check condition could be useless like if the `relativeLayout` will be `null` the system already through a `ClassCastException` because null cannot be cast so null check must be added before the casting.